components/clarity-repl/src/repl/session.rs

```diff
@@ -963,7 +963,15 @@ impl Session {
         let accounts = self.interpreter.get_accounts();
         if accounts.len() > 0 {
             let tokens = self.interpreter.get_tokens();
-            let headers = vec!["Address".to_string(), "uSTX".to_string()];
+            let mut headers = vec!["Address".to_string()];
+            for token in tokens.iter() {
+                if String::from("STX").eq(&String::from(token)) {
```

Comment:
You should be able to do `if token == "STX"` here, or is their any reason why not?

Reply:
The operator `==` relies on the `eq` function to do the comparison. Thus, this add one more underlying function call to the stack. But I think there is no need to worry with that... it's only a "muscular" memory.

And I should agree that the infix notation (1 == 1) is more readable. :-)... changed.

Reply:
Oh okay, didn't know about that, thanks for the explanation